react/ProductGiftName.tsx

```diff
@@ -62,6 +76,12 @@ ProductGiftName.schema = {
       description: messages.linkToProductDescription.id,
       type: 'boolean',
     },
+    nameType: {
+      default: false,
+      title: messages.nameType.id,
+      description: messages.nameType.id,
+      type: 'boolean',
+    },
```

Comment:
This definition would result in a boolean field in the Site Editor. I think you meant to declare an enum here, right?

Reply:
Here's an example: https://github.com/vtex-apps/menu/blob/master/store/contentSchemas.json#L63

Reply:
Yes it was my mistake, I fixed it.
@victorhmp